### PR TITLE
change capitalization of daemon name

### PR DESCRIPTION
**Pull request by kbhawkey**
Update NPD daemon name to sentence case (Node Problem Detector).

### CODE REVIEW

**Comment by kbhawkey** [comment]
Related to #24772.

**Review by Aut0R3V — APPROVED** [review]

**Review by ydFu — APPROVED** [review]

**Comment by kbhawkey** [comment]
@karan , Would you have time to look over these changes?

**Comment by karan** [comment]
/lgtm

Thanks for the follow up

File: content/en/docs/tasks/debug-application-cluster/monitor-node-health.md [hunk 1]
```diff
@@ -8,14 +8,14 @@ reviewers:
 
 <!-- overview -->
 
-*Node problem detector* is a daemon for monitoring and reporting about a node's health.
-You can run node problem detector as a `DaemonSet`
-or as a standalone daemon. Node problem detector collects information about node problems from various daemons
+*Node Problem Detector* is a daemon for monitoring and reporting about a node's health.
```
Comment (bradtopol) [inline content/en/docs/tasks/debug-application-cluster/monitor-node-health.md:11]: Great job!

**Comment by bradtopol** [comment]
Great job! Verified that new updates are correct and render properly!
/lgtm
/approve

**Pull request merged**